Core/Lib/Export/PDFExport.php

```diff
@@ -227,6 +231,8 @@ protected function insertBusinessDocBody($model)
             $tableData[$key]['price'] = $this->numberTools->format($value['price']);
             $tableData[$key]['discount'] = $this->numberTools->format($value['discount']);
             $tableData[$key]['tax'] = $this->numberTools->format($value['tax']);
+            $tableData[$key]['totalSurcharge'] = $this->numberTools->format($value['totalSurcharge']);
```

Comment:
$tableData[$key]['surcharge'] = $this->numberTools->format($value['surcharge']);